pytest_fixtures/component/http_proxy.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from fauxfactory import gen_string
+
+from robottelo.config import settings
+
+
+def create_http_proxy(sat, org, http_proxy_type):
```

Comment:
I'm not understanding the use of `sat` and `org` here as opposed to `target_sat` and `module_org`.  Can you explain?

Reply:
It's just a helper function, which is used by the `function_http_proxy` fixture.